test/hooks/views/res.render.i18n.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * Module dependencies
+ */
+
+var assert = require('assert');
+var Sails = require('../../../lib').constructor;
+var path = require('path');
+
+describe('res.render() with i18n', function (){
```

Comment:
@Josebaseba minor thing: can we change the "describe" label to say `sails.hooks.views.render()` instead of `res.render`?